### PR TITLE
Release Notes for 3/10/15

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@catong   @gwprice 

@explorerleslie  - anything to add in Coming Soon (we removed Google docs)

@cptvitamin  - many accessibility changes in studio/lms pages.

File: en_us/release_notes/source/xblocks/index.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@ edX XBlocks
 ####################################
 
 ******************
+March 10, 2015
+******************
+
+A new XBlock service allows XBlocks to acces Django settings. (OSPR-419,
```
Comment (catong) [inline en_us/release_notes/source/xblocks/index.rst]: Missing "s" at the end of "access"

File: en_us/release_notes/source/insights/index.rst [hunk 2]
```diff
@@ -3,6 +3,16 @@ edX Insights
 ####################################
 
 ******************
+March 10, 2015
+******************
+
+Course teams can now review how students are answering graded problems in edX
```
Comment (catong) [inline en_us/release_notes/source/insights/index.rst]: Possibly reads as if students are answering problems in edX Insights. Recommend moving that to the beginning of sentence: "In edX Insights, course teams can now review..."

**Comment by catong** [comment]
:+1: with a couple of minor nits. If there are further additions I can proofread again, otherwise looks fine.

File: en_us/release_notes/source/documentation/index.rst [hunk 3]
```diff
@@ -4,6 +4,21 @@ edX Documentation
...
+March 10, 2015
+******************
+
+==================================
+edX Research Guide
+==================================
+
+The `Tracking Logs`_ chapter now contains:
+
+* New events for contributions to discussion forums.
```
Comment (explorerleslie) [inline en_us/release_notes/source/documentation/index.rst:16]: Wasn't this included in last week's release? (Apologies if I'm losing it...)

File: en_us/release_notes/source/studio/index.rst [hunk 4]
```diff
@@ -3,6 +3,22 @@ edX Studio
...
 ******************
+March 10, 2015
+******************
+
+============================
+Accessibility Improvements 
+============================ 
+
+* The **Schedule & Details**, **Group Configurations**, **Advanced Settings**,
+  and **Grading** pages now include `aria-label` attributes. (TNL-1532)
```
Comment (explorerleslie) [inline en_us/release_notes/source/studio/index.rst:14]: @cptvitamin yay!

File: en_us/release_notes/source/studio/index.rst [hunk 5]
```diff
@@ -3,6 +3,22 @@ edX Studio
...
+
+* All page actions in Studio now have `aria-label` attributes. (TNL-1531)
```
Comment (explorerleslie) [inline en_us/release_notes/source/studio/index.rst:16]: @cptvitamin double yay!

File: en_us/release_notes/source/studio/index.rst [hunk 6]
```diff
@@ -3,6 +3,22 @@ edX Studio
...
+  
+* Navigation controls in the Studio header now have `aria-label` attributes.
+  (TNL-1520, TNL-1523)
```
Comment (explorerleslie) [inline en_us/release_notes/source/studio/index.rst:19]: @cptvitamin triple yay!

**Comment by explorerleslie** [comment]
@mhoeber one minor question on whether forum events were released this week or last week. otherwise :+1:

File: en_us/release_notes/source/documentation/index.rst (hunk 3 above)
Comment (mhoeber) [inline en_us/release_notes/source/documentation/index.rst:16]: it was done on the 5th, same day as release, but didn't get listed here.

**Comment by cptvitamin** [comment]
:+1: This all looks great to me :)

**Pull request merged**